pkg/utils/parse_arf_result.go

```diff
@@ -552,12 +569,29 @@ func getSafeText(nptr *xmlquery.Node, elem string) string {
 	return elemNode.InnerText()
 }
 
-func complianceCheckResultDescription(rule *xmlquery.Node) string {
+func complianceCheckResultDescription(rule *xmlquery.Node, valuesList map[string]string) (string, error) {
 	title := getSafeText(rule, "xccdf-1.2:title")
 	if title != "" {
 		title = title + "\n"
 	}
-	return title + getSafeText(rule, "xccdf-1.2:rationale")
+	description, err := getElementText(rule, "xccdf-1.2:description", valuesList)
+	return title + description, err
+}
+
+func complianceCheckResultRationale(rule *xmlquery.Node, valuesList map[string]string) (string, error) {
+	return getElementText(rule, "xccdf-1.2:rationale", valuesList)
+}
+
+func getElementText(nptr *xmlquery.Node, elem string, valuesList map[string]string) (string, error) {
+	ele := nptr.SelectElement("xccdf-1.2:rationale")
```

Comment:
This should say elem, right?

Reply:
ahh nice catch, don't know how did I miss that